scripts.v3/utils.js

```diff
@@ -88,7 +87,7 @@ class HttpClient {
                             reject({ code: "Forbidden", message: `Looks like you are not allowed to perform this operation. Please check with your administrator.` });
                             break;
                         default:
-                            reject({ code: "UnhandledError", message: `Could not complete request to ${requestUrl}. Status: ${resp.statusCode} ${resp.statusMessage}` });
+                            reject({ code: "UnhandledError", message: `Could not complete request to ${requestUrl}. Status: ${resp.statusCode} ${resp.statusMessage}, Data: ${data}` });
```

Comment:
Just checking if any sensitive data can be logged here. It might be a lot of data logged there as well, no?